src/wp-includes/rest-api/endpoints/class-wp-rest-font-collections-controller.php

```diff
@@ -0,0 +1,332 @@
+<?php
+/**
+ * Rest Font Collection Controller.
+ *
+ * This file contains the class for the REST API Font Collection Controller.
+ *
+ * @package    WordPress
+ * @subpackage Font Collection
+ * @since      6.4.0
+ */
+
+/**
+ * Font Collection Controller class.
+ *
+ * @since 6.4.0
+ */
+class WP_REST_Font_Collections_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 6.4.0
+	 */
+	public function __construct() {
+		$this->rest_base = 'font-collections';
+		$this->namespace = 'wp/v2';
+	}
+
+	/**
+	 * Registers the routes for the objects of the controller.
+	 *
+	 * @since 6.4.0
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_font_collections' ),
+					'permission_callback' => array( $this, 'update_font_library_permissions_check' ),
+				),
+				'schema' => array( $this, 'get_items_schema' ),
```

Comment:
There idea of schema, is the desciptions an item in the response in the request. The shape of the repsonse should match between items and item endpoints. Items should just be a array of item. That way you can reuse the same prepare_items_for_response. See this example. 

https://github.com/WordPress/wordpress-develop/blob/23c811e59fe3a634800e3672df7dc8d45556ae6a/src/wp-includes/rest-api/endpoints/class-wp-rest-block-types-controller.php#L133-L157